src/io/fd.cpp

```diff
@@ -75,6 +75,10 @@ namespace io
     {
         return ::write(fd, buf, count);
     }
+    ssize_t FD::send(const void *buf, size_t count)
```

Comment:
This is an abuse of the `io::FD` interface.

Any methods on `io::FD` must correspond *exactly* except for the type of the zeroeth argument. Passing a specific flag should be the responsibility of the caller of `io::FD::send`.